### PR TITLE
added CONTRIBUTING.md in Indonesian (CONTRIBUTING-id.md)

### PR DESCRIPTION
**Pull request by reski-mulud-muchamad**
## What does this PR do?
Add resource(s)

### Description

I added CONTRIBUTING.md into the Indonesian translation.

### Why is this valuable (or not)?

this will be useful if there are Indonesians who want to contribute to this repository, then they will have no difficulty in reading the contributing guide

## Checklist:
- [✓] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [✓] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Sorry about conflicts, we're merging oldest PRs first

**Comment by eshellman** [comment]
I would separate the translation and the links to it into two separate PRs. The links generate the conflicts, and we can do a cleanup PR after things have settled a bit.

File: CONTRIBUTING-id.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+    - *Buku* : PDF, HTML, ePub, situs berbasis gitbook.io, repo Git, dll.
+    - *Kursus* : Kursus adalah bahan ajar yang bukan buku. [Ini adalah kursus](http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-006-introduction-to-algorithms-fall-2011/).
+    - *Tutorial Interaktif* : Situs web interaktif yang memungkinkan pengguna mengetik kode atau perintah dan mengevaluasi hasilnya (dengan "mengevaluasi" kami tidak bermaksud "menilai"). misalnya.: [Try Haskell](http://tryhaskell.org), [Try Github](http://try.github.io).
+    - *Podcast dan Screencasts* : Podcast dan screencast.
+    - *Rangkaian Masalah & Pemrograman Kompetitif* : Situs web atau perangkat lunak yang memungkinkan Anda menilai keterampilan pemrograman Anda dengan memecahkan masalah sederhana atau kompleks, dengan atau tanpa tinjauan kode, dengan atau tanpa membandingkan hasilnya dengan pengguna lain.
+
+4. Pastikan untuk mengikuti [panduan di bawah](#guidelines) dan hormati [Pemformatan file Markdown](#formatting).
+
+5. GitHub Actions akan menjalankan tes untuk memastikan daftar Anda diurutkan menurut abjad dan aturan pemformatan diikuti. Pastikan untuk memeriksa bahwa perubahan Anda lulus tes.
```
Comment (davorpa) [inline CONTRIBUTING-id.md:23]: Satisfies #5644

File: CONTRIBUTING-id.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+Kursus sering kali memiliki buku teks terkait, yang akan kami daftarkan dalam daftar buku kami. Kursus memiliki kuliah, latihan, tes, catatan atau alat bantu didaktik lainnya. Sebuah kuliah atau video dengan sendirinya bukanlah sebuah kursus. Sebuah powerpoint bukanlah kursus.
+
+##### Tutorial Interaktif vs. Hal-hal lain
+
+Jika Anda dapat mencetaknya dan mempertahankan esensinya, itu bukan Tutorial Interaktif.
+
+
+### Otomatisasi
+
+- Pemformatan penegakan aturan otomatis melalui [GitHub Actions](https://github.com/features/actions) gunakan [fpb-lint](https://github.com/vhf/free-programming-books-lint) (lihat [.github/workflows/fpb-lint.yml](.github/workflows/fpb-lint.yml))
```
Comment (davorpa) [inline CONTRIBUTING-id.md:205]: Satisfies #5644

**Comment by eshellman** [comment]
remove your changes to the other language files, that will fix the conflicts.

**Comment by davorpa** [comment]
No deletions. Only a hard reset to previous state 😉 for this files

**Comment by davorpa** [comment]
Stale.

@reski-mulud-muchamad Could you resolve conflicts? If context is needed, tell us!

**Comment by eshellman** [comment]
added in #5816

**Comment by davorpa** [comment]
> added in #5816

Better... this complete #5816 translation, I mention in the backlog #5762. To don't deal with conflicts I'll pick up it in the sync PR that I'm working

**Pull request closed without merging**